docs/rfcs/1000-shamir-based-recovery.md

```diff
@@ -233,6 +232,8 @@ Consistency between `brief` and `shares` must be checked by the Parsec server:
 > organization-specific rules (e.g. the number of shares, the recipient's
 > profiles, max number of share per recipient, etc.). See "Bonus" section below.
 
+To update the setup, the previous setup must be deleted first.
```

Comment:
On top of that, the `shamir_setup_already_exists` should have a `DateTime` field `last_shamir_certificate_timestamp`.

This is important given the Parsec client is expected to eagerly download all certificates, hence it should have been aware of the already existing shamir setup. And if that's not the case it is because the client is lagging behind (typically the certificate is very new), and in this case the client polls the server for certificates... but there is great chances this is already under way (the certificates ops start polling as soon as it gets the "new certificates" events from the server), hence this `last_shamir_certificate_timestamp` that allows to detect if the poll is really needed or if it has been concurrently achieved.

See for instance what is done for `user_revoke` command:
https://github.com/Scille/parsec-cloud/blob/048b4c6528cbb1b78b6daa9d567f356c6307462f/libparsec/crates/protocol/schema/authenticated_cmds/user_revoke.json5#L26-L31
and in the client (note the `RemoteIdempotent` status that means the server rejected our certificate, but we are missing some certificates and once we fetch them we will discover our current operation has already been achieved):
https://github.com/Scille/parsec-cloud/blob/048b4c6528cbb1b78b6daa9d567f356c6307462f/libparsec/crates/client/src/client/user_revoke.rs#L15-L26

Reply:
> On top of that, the shamir_setup_already_exists should have a DateTime field last_shamir_certificate_timestamp. 

Done in #7622 :tada: